dds/DCPS/MultiTopicDataReaderBase.cpp

```diff
@@ -50,10 +50,8 @@ namespace {
       try {
         outer_->data_available(reader);
       } catch (std::exception& e) {
-        if (OpenDDS::DCPS::DCPS_debug_level) {
-          ACE_ERROR((LM_ERROR, "(%P|%t) MultiTopicDataReaderBase::Listener::"
-                     "on_data_available(): %C", e.what()));
-        }
+        ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) ERROR: MultiTopicDataReaderBase::Listener::on_data_available: %C\n"),
```

Comment:
This changed debug level -- was that intended?

Reply:
For this particular one I'm not sure. I know the other ones were hiding errors in the SQL and I didn't think that was right.